x-pack/plugins/secops/public/components/timeline/body/actions/index.tsx

```diff
@@ -76,47 +57,57 @@ export const Actions = pure<Props>(
     toggleShowNotes,
     updateNote,
   }) => (
-    <ActionsContainer data-test-subj="timeline-actions-container">
-      <ActionsRows data-test-subj="timeline-actions-rows">
-        <ActionsRow data-test-subj="timeline-actions-row">
-          <EuiButtonIcon
-            aria-label={expanded ? i18n.COLLAPSE : i18n.EXPAND}
-            color="text"
-            iconType={expanded ? 'arrowDown' : 'arrowRight'}
-            data-test-subj="timeline-action-expand"
-            id={eventId}
-            onClick={onEventToggled}
-          />
-          <EuiToolTip
-            data-test-subj="timeline-action-pin-tool-tip"
-            content={getPinTooltip({
-              isPinned: eventIsPinned,
-              eventHasNotes: eventHasNotes(noteIds),
-            })}
-          >
-            <PinContainer>
-              <Pin
-                allowUnpinning={!eventHasNotes(noteIds)}
-                pinned={eventIsPinned}
-                data-test-subj="timeline-action-pin"
-                onClick={onPinClicked}
-              />
-            </PinContainer>
-          </EuiToolTip>
-          <NotesButton
-            animate={false}
-            associateNote={associateNote}
-            data-test-subj="timeline-action-notes-button"
-            getNotesByIds={getNotesByIds}
-            noteIds={noteIds || emptyNotes}
-            showNotes={showNotes}
-            size="s"
-            toggleShowNotes={toggleShowNotes}
-            toolTip={i18n.NOTES_TOOLTIP}
-            updateNote={updateNote}
-          />
-        </ActionsRow>
-      </ActionsRows>
+    <ActionsContainer
+      alignItems="flexStart"
+      data-test-subj="timeline-actions-container"
+      direction="row"
+      gutterSize="none"
+      justifyContent="spaceBetween"
+    >
+      <EuiFlexItem>
+        <EuiButtonIcon
+          aria-label={expanded ? i18n.COLLAPSE : i18n.EXPAND}
```

Comment:
Is this right? if expanded show the `i18n.COLLAPSE` vs `EXPAND`? Also ensure that the Aria Labels are marked in the `i18n` file as having a suffix if `AriaLabel` per i18n guidelines:

https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types

Reply:
I updated the `i18n` file to include the `AriaLabel` suffix.

Regarding your question:
> Is this right? if expanded show the `i18n.COLLAPSE` vs `EXPAND`?

The `aria-label` is describing the action that will occur if the user clicks the button. That action is in fact, the opposite of the current state.